src/EFCore.PG/Scaffolding/Internal/NpgsqlDatabaseModelFactory.cs

```diff
@@ -516,15 +516,6 @@ ELSE pg_get_expr(indpred, cls.oid)
 
                         if (columnIndices.Any(i => i == 0))
                         {
-                            var expressions = record.GetValueOrDefault<string>("exprs");
-
-                            if (expressions?.StartsWith("to_tsvector(") == true)
-                            {
-                                ConfigIndexBaseOnTsVectorExpression(expressions, tableColumns, index);
-                                table.Indexes.Add(index);
-                                continue;
-                            }
-
```

Comment:
Why is this removed?

Reply:
See https://github.com/npgsql/efcore.pg/pull/1181#discussion_r377755473